lerobot/common/datasets/lerobot_dataset.py

```diff
@@ -884,6 +887,87 @@ def add_frame(self, frame: dict) -> None:
 
         self.episode_buffer["size"] += 1
 
+    def add_episode_to_batch(self, episode_data: dict | None = None) -> None:
+        if episode_data is None:
+            episode_data = self.episode_buffer
+        self.episode_batch.append(episode_data)
+        episode_idx = self.meta.total_episodes + len(self.episode_batch)
+        self.episode_buffer = self.create_episode_buffer(episode_index=episode_idx)
+
+    def save_episode_batch(self, episode_data: dict | None = None) -> None:
```

Comment:
Is this just a repeat of save_episode with a loop at the top? Can we instead call save_episode instead of duplicating this logic?